biz.aQute.resolve/src/biz/aQute/resolve/Bndrun.java

```diff
@@ -161,6 +165,23 @@ public int compare(VersionedClause a, VersionedClause b) {
 	}
 
 	/**
+	 * Copy the calculated <code>-runbundles</code> into a directory.
+	 *
+	 * @param targetDir the directory in which <code>-runbundles</code> are
+	 *            placed
+	 * @throws Exception
+	 */
+	public void runbundles(File targetDir) throws Exception {
```

Comment:
Fixed to use the project method.

Also fixed the issue for potential file name collisions. 
However, fixing the issue for the launcher required slightly different code which I couldn't centralize in one place because the API for talking to file systems and jars wasn't close enough to warrant it. But, I did solve the issue, but in the simplest way possible. I added a _Windows_ style renaming logic.